lib/apiservers/engine/backends/archive.go

```diff
@@ -447,7 +442,7 @@ func (wm *ArchiveStreamWriterMap) WriterForAsset(proxy proxy.VicArchiveProxy, ci
 			deviceID = aw.mountPoint.Name
 			store = constants.VolumeStoreName
 		}
-		rawWriter, err := proxy.ArchiveImportWriter(wm.op, store, deviceID, aw.filterSpec)
+		rawWriter, err := proxy.ArchiveImportWriter(wm.op, store, deviceID, aw.filterSpec, wm.errchan)
```

Comment:
I've mainly seen channels that return data returned from functions, but passing one in works too.  I'm not sure if there is any idiomatic rule about this.  Just a comment.